tests/playwright/src/specs/podman-machine-dashboard-onboarding.spec.ts

```diff
@@ -62,9 +62,10 @@ describe.skipIf(os.platform() === 'linux')(async () => {
       test('Create Podman machine from Dashboard', async () => {
         const navigationBar = new NavigationBar(page);
         const dashboardPage = await navigationBar.openDashboard();
-        await playExpect(dashboardPage.initilizeAndStartButton).toBeEnabled();
-        await dashboardPage.initilizeAndStartButton.click();
-        await playExpect(dashboardPage.podmanMachineConnectionStatus).toHaveText('RUNNING', { timeout: 300000 });
+        await playExpect(dashboardPage.podmanProvider).toBeVisible();
```

Comment:
Why is this assertion needed? The initialize and start button is a component of that component, if we're waiting for it to be enabled it stands to reason that the higher level component exists also. IS there some race condition this asset solves that I'm not seeing?

Reply:
not really. I like to start asserting from parent elements.
Ie.: Is a dialog opened? No? it does not make sense to press "Next" button and I know exactly that wizard did not appear. 
In this case the error (if by some coincidence it would happen) would just say Timeout Waiting for locator to get visible:
```
Error: Timed out 5000ms waiting for expect(locator).toBeVisible()

Locator: getByRole('navigation', { name: 'PreferencesNavigation' }).getByRole('link', { name: 'Red Hat OpenShift Local' })
```
And it is not really precise as of which of locators was faulty here. Since it chains it together. Does it make sense?

Reply:
No, I'm not following, what does that code snippet have to do with anything?

The point is that it just adds more lines of code to the test that don't have real practical purpose but on the other hand increase potential failure points. I would agree with you that checking for the parent element makes sense if we would be interacting with it in some way, but we don't, we only interact with the button in it as such it makes no real sense to check the parent element visibility.

If we were to check the visibility of every parent element it would mean we'd have to make ten asserts until we reach the button. It's just code bloat.

Reply:
OpenShift Local has the same button and the same functionality. In case that you have both or just openshift local, you would be clicking on totally different element.
since I agree with not checking the parent a locator involving getByRole('region', 'Podman Provider') must be there so we are sure are clicking right button.

Reply:
Yes. As I said, with the adding a provider distinction we could omit it.